xml/obs_multibuild.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter PUBLIC
+  "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.docbook.org/xml/4.5/docbookx.dtd"
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+  %entities;
+]>
+<chapter id="cha.obs.multibuild">
+ <title>Multibuild</title>
+ <section>
+  <title>Overview</title>
+  <para>
+   Multibuild is used for those sources that can be used to build more than a
+   package. For that to be accomplished you must provide a file with a list of
+   package names. For each package name of that list it will be built a
+   different set of package builds (with their repositories and architectures).
```

Comment:
I think its simpler to say:

"A set of package builds, repositories and architectures will be built for each package in that list."